registry/corestake/calldata-corestake.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://eips.ethereum.org/assets/eip-7730/erc7730-v1.schema.json",
+  "context": {
+    "contract": {
+      "abi": "https://openapi.coredao.org/api/contracts/abi_of_verified_contract/0xf5fA1728bABc3f8D2a617397faC2696c958C3409",
+      "deployments": [
+        {
+          "chainId": 1116,
+          "address": "0xf5fA1728bABc3f8D2a617397faC2696c958C3409"
+        }
+      ]
+    }
+  },
+  "metadata": {
+    "name": "CoreStake",
```

Comment:
The CI is failing because of the metadata object. It should look like

```
"metadata": {
  "owner": "CoreStake",
  "info": {
    "legalName": "CoreStake Staking Contract",
    "url": "https://coredao.org/stake",
    "deploymentDate": "<contract deployement date>"
  }
}
```